### PR TITLE
Enforce prettier in CI

### PR DESCRIPTION
**Pull request by PowerKiKi**
To be merged only after #1801

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -21,6 +21,7 @@ jobs:
           node-version: 18
       - run: npm ci
 
+      - run: ./node_modules/.bin/prettier --check .
```
Comment (santam85) [inline .github/workflows/ci.yml:24]: Please use `npx prettier . --check` or `npm run prettier` with a custom script in package.json instead

Comment (PowerKiKi) [reply .github/workflows/ci.yml]: Done in d3b6f054569686ce2132b7ba545fb4c348897233

**Comment by PowerKiKi** [comment]
CI should be passing and be ready to merge

**Comment by PowerKiKi** [comment]
rebased

**Comment by santam85** [comment]
Should we add a pre-commit hook as well? https://prettier.io/docs/en/install#git-hooks

**Pull request merged**